website/docs/implementation_guides/maintainer/readme.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Introduction
+sidebar_label: Introduction
+---
+
+## Introduction
+
+- Who it's for
+- Ecosystem view: How it all hangs together (FFI, plugins, client language, CLI tools. Possibly use the diagrams from docs.pact.io/plugins)
+- Maintainer meetings (how to stay in touch. Slack, Zoom monthly etc.)
+- Roadmap (Where to find it, how to contribute to it, RFC process)
+- Criteria for an implementation to be accepted into pact-foundation
+  - License should be MIT
```

Comment:
Also Apache2

Reply:
yeah we have a handful of Apache 2.0 repos.

I noted about 9 don't have licenses, which probably should so will flag them for review.